nsqlookupd/registration_db.go

```diff
@@ -230,6 +247,18 @@ func (pp Producers) PeerInfo() []*PeerInfo {
 	return results
 }
 
+func (pp Producers) IsPaused() bool {
+	pausedCount := 0
+	total := 0
+	for _, p := range pp {
+		if p.paused == true {
+			pausedCount++
+		}
+		total++
+	}
+	return pausedCount*2 > total
```

Comment:
`>=`? I think in the two producer case, if one is paused we probably want to know that.

Reply:
done.